packages/simulator/web/routes.js

```diff
@@ -116,22 +115,26 @@ function register(io) {
 
           const averageUtilization = cars.pipe(
             mergeMap(car => fromEvent(car, 'cargo')),
-            windowTime(15000),
-            map(window => window.pipe(
-              distinct(car => car.id),
-              scan(({ totalCargo, totalCapacity, totalQueued }, { cargo, queue, capacity }) => ({
-                totalCargo: totalCargo + cargo.length,
-                totalCapacity: totalCapacity + capacity,
-                totalQueued: totalQueued + queue.length
-              }), { totalCargo: 0, totalCapacity: 0, count: 0, totalQueued: 0 }),
-              map(({ totalCargo, totalCapacity, totalQueued }) => ({ 
-                totalCargo, totalCapacity, totalQueued, 
-                averageUtilization: totalCargo / totalCapacity, 
-                averageQueued: totalQueued / totalCapacity 
-              }))
-            )),
-            mergeAll(),
-            startWith({ totalCargo: 0, totalCapacity: 0, averageUtilization: 0, averageQueued: 0}),
+            scan((acc, car) => {acc[car.id] = car; return acc}, {}),
```

Comment:
I don't see how we can keep an average without keeping track of individual cars. So this creates a state of `{[car.id] => state}` which is then used to sum and average